recipes/diligent-core/all/test_v1_package/conanfile.py

```diff
@@ -0,0 +1,17 @@
+from conans import ConanFile, CMake, tools
```

Comment:
test_v1_package is not needed anymore, please leave only test_package files and the cpp there